### ISSUE TITLE
Prenatal (CPN) report issues

### ISSUE THREAD

**Issue by space-ravi** [kamal1978/LTFHC#293]
Some calculation issues have been discovered in the CPN report: 

1. CPN 4 (Prenatal Care 4) should increment every time:
-- English: Component 237 is "4th" or "4+"
-- French: Component 237 is "CPN4" or "CPN4+" 

2. CPN 4 (Prenatal Care 4) at the 36th week should increment every time: 
-- English: (Component 237 is "4th" or "4+") AND (Component 220 is "9 months")
-- French: (Component 237 is "CPN4" or "CPN4+") AND (Component 220 is "9 mois")

3. Pregnant women with PB < 230mm should increment every time: 
-- English: Component 214 is < 230
-- French: Component 214 is < 230

4. Mosquito-net MILD distributed during CPN 1 should increment every time: 
-- English: Component 344 is "Yes" AND Component 237 is "1st"
-- French: Component 344 is "Oui" AND Component 237 is "CPN1"

5. Mosquito-net MILD distributed during CPN 2+ should increment every time: 
-- English: Component 344 is "Yes" AND (Component 237 is "2nd" OR "3rd" OR "4th" OR "4+")
-- French: Component 344 is "Oui" AND (Component 237 is "CPN2" OR "CPN3" OR "CPN4" OR "CPN4+")

**Comment by space-ravi** [comment]
Everything worked EXCEPT the "Mosquito-net MILD distributed during CPN 2+" calculation:
- I entered a single patient, CPN1 visit (1st), and Yes received MILD. In the report, it showed "1" next to both "Mosquito-net MILD distributed during CPN 1" (expected) and "Mosquito-net MILD distributed during CPN 2+" (wrong).

- When I entered CPN2 visit (2nd), and Yes received MILD, the report did not increment any rows (it should have incremented the "Mosquito-net MILD distributed during CPN 2+" entry.
